src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

```diff
@@ -510,6 +510,70 @@ g.test('copy_within_same_texture')
     );
   });
 
+g.test('copy_aspects')
+  .desc(
+    `
+Test the validations on the member 'aspect' of GPUTextureCopyView in CopyTextureToTexture().
+- for all the color and depth-stencil formats: the texture copy aspects must be both 'all'.
+- for all the depth-only formats: the texture copy aspects must be either 'all' or 'depth-only'.
+- for all the stencil-only formats: the texture copy aspects must be either 'all' or 'stencil-only'.
+`
+  )
+  .params(
+    params()
+      .combine(
+        poptions('format', [
+          'rgba8unorm',
+          'depth24plus-stencil8',
+          'depth32float',
+          'stencil8',
+        ] as const)
```

Comment:
nit: May as well test every depth/stencil format in kDepthStencilFormats, in addition to rgba8unorm. Something like:

```suggestion
        poptions('format', [
          'rgba8unorm',
          ...kDepthStencilFormats,
        ] as const)
```